documentation/docs/instructor.md

```diff
@@ -1,82 +1,99 @@
+Navigate to https://cloudvcl.cci.drexel.edu/ and log in using your Drexel credentials. You will know that you've successfully logged in when you see `Logged in as YourUsername` in the top right corner. Once you log in and arrive on the home page, if you see a blue text box that indicates you're an instructor, you've been successfully promoted. If not, please ask your system administrator to promote you. To log out, select the `Logout` button in the top right corner.
+
+## Checking Quota and Usage
+Whenever you or your student creates an instance, it will be indicated in the blue text box along with other usage details (CPU, RAM, and disk).  
+
+`Reminder: Your allocated quota is shared among all your students in all your courses.`
+
+## Creating a Course 
+Visit the Course List page by selecting the `Courses` tab found in the header. Create a VCL course by finding and clicking the `Create Course` button near the top right corner, providing a name for the desired course, and pressing `Submit`. Courses can be edited or deleted on the Course List page. 
+
+## Adding Students to a Course (CSV Upload)
+Click on a course title on the Course List page and then press the `Add Students (CSV Upload)` button. Upload a CSV file with proper fields `(Last Name, First Name, Username)`. Once uploaded, the data will be parsed and saved to the Cloud VCL database and the students listed in the CSV will be added to the course.
+
+`Reminder: The format of the CSV file used by Blackboard Learn may have additional fields; however, Cloud VCL only requires (Last Name, First Name, Username)`.  
+
+## Creating an Environment Definition
+The environment definition is a virtual environment where assignments will be created through defining VMs. View the Environment Definition List page by selecting the `Environment Definitions` tab found in the header. Create an environment definition by clicking `Create Environment Definition` in the top right corner and providing a name. Click on a environment definition title to view any corresponding defined VMs. Environment definitions can be edited or deleted through the Environment Definition List page. 
+
+`You can create multiple Environment Definitions for one Assignment, which can hold one or more VM Definitions.`
+
+## Creating a VM Definition
+By clicking on an environment definition title from the Environment Definition List page, you can view, create, edit, or delete  the Virtual Machine Definitions that have been created. Create a new VM by clicking 'Create VM Definition` in the top right corner. These VM Definitions will be used to load virtual machines into the environment when it's launched through an assignment. Find more details about specific fields in the creation of a VM below:  
+
+### Image 
+Choose the type of image you wish to use: CentOS 7.3, Ubuntu Server 16.04.2, Windows Server 2012 R2
+### Flavor
+Select the type of flavor: 01c-01m-10d reads 1 VCPU, 1 RAM GB, and 10 GB Total/Root Disk.
```

Comment:
That's wrong, this only applies to our cloud, other clouds may have completely different flavors and flavor names.